core/src/test/java/org/apache/calcite/test/MaterializationTest.java

```diff
@@ -126,6 +126,13 @@
       new RexSimplify(rexBuilder, RelOptPredicateList.EMPTY, RexUtil.EXECUTOR)
           .withParanoid(true);
 
+  private final String deduplicatedEmps =
+      "(select distinct \"empid\", \"deptno\", \"name\", \"salary\", \"commission\"\n"
+          + "from \"emps\") \"dedupEmps\"";
+  private final String deduplicatedDepts =
+      "(select distinct \"deptno\", \"name\"\n"
+          + "from \"depts\") \"dedupDepts\"";
```

Comment:
Although it make duplicate query string less, it will make us hard to read. We have to scroll up to see what exactly the subquery is. I would recommend inlining them, and remove these 2 variables.

Reply:
But it is up to you. Not a requirement.